lib/mix/test/mix/tasks/profile.fprof_test.exs

```diff
@@ -53,45 +59,50 @@ defmodule Mix.Tasks.Profile.FprofTest do
   test "sort options", context do
     in_tmp context.test, fn ->
       assert capture_io(fn ->
-        Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)", "--sort", "acc"])
-      end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
+               Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)", "--sort", "acc"])
+             end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
 
       assert capture_io(fn ->
-        Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)", "--sort", "own"])
-      end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
+               Fprof.run(["-e", "Enum.each(1..5, fn(_) -> MapSet.new end)", "--sort", "own"])
+             end) =~ ~r(MapSet\.new/0 *5 *\d+\.\d{3} *\d+\.\d{3})
     end
   end
 
   test "errors on missing files", context do
     in_tmp context.test, fn ->
-      assert_raise Mix.Error, "No files matched pattern \"non-existent\" given to --require", fn ->
-        capture_io(fn -> Fprof.run ["-r", "non-existent"] end)
-      end
-
-      assert_raise Mix.Error, "No files matched pattern \"non-existent\" given to --require", fn ->
-        capture_io(fn -> Fprof.run ["-pr", "non-existent"] end)
-      end
+      assert_raise Mix.Error,
```

Comment:
Please format this and below as in example #3 here: https://github.com/elixir-lang/elixir/issues/6643

Reply:
sure....